src/libstore/derivation-options.hh

```diff
@@ -52,6 +65,8 @@ struct DerivationOptions
 
     OutputChecks checksAllOutputs;
 
+    AdditionalAttributes attrs;
+
```

Comment:
I don't think this should go here, but replacing `env` in `BasicDerivation`

Reply:
Ah yes oopsie, should it still be named `env` though?

Reply:
name doesn't matter, since call sites need to be updated either way.

Yes probably should go there.